### DISCUSSION TITLE
Question about global variables

### DISCUSSION THREAD

**Discussion by Amumu2020** [duckdb/duckdb#7731] [General]
Assuming I have a database named mydb, I will do the following:
1. Connect to mydb and modify the global variable like this: `set global threads = 4;`
2. Exit Connection
3. Reconnect to mydb and view the current variable value: `select current_setting('threads');`

Does it still return the default value (16) as expected?

**Comment by Mause** [comment]
No, settings don't persist between sessions presently

**Reply by Amumu2020** [reply]
Thank you very much.

If global variables don't persist between sessions, what is the difference between them and the session variables?

![1](https://github.com/duckdb/duckdb/assets/26343224/c91536fe-843e-4dcd-8700-fd28bad850a8)

**Reply by Mause** [reply]
Global refers to being shared between multiple connections in a single process

**Reply by Amumu2020** [reply]
I got it.  Thank you again for your answer!
